app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,127 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+
+		if logged_in?
+			@tweets = Tweet.all
+
+			erb :'tweets/tweets'
+		else
+			redirect to '/login'
+		end
+	end
+
+
+	post '/tweets' do
+		if logged_in?
+			if params[:tweet] == ""
+				redirect to "/tweets/new"
+			else
+				Tweet.all.clear
+				@user = User.find_by(email: session[:user_id][:email])
+				@user.tweets << Tweet.create(content: params[:tweet])
+				
+				redirect to '/tweets'
+			end
+		end
+
+	end
+
+
+
+
+	get '/tweets/new' do 
+
+
+		if logged_in?
+			erb :'tweets/create_tweet'
+		else
+			redirect to "/login"
+		end
+
+	end
+
+	get '/tweets/:id/edit' do
+
+		if logged_in?
+			@user = current_user
+			@tweet = Tweet.find_by(id: params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?